airflow/api/common/experimental/get_dags.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Dags APIs.."""
+import typing
+
+from airflow.api.common.experimental import check_and_get_dags
+
+
+def get_dags() -> typing.List[str]:
+    """Return a list of ids of the dags located within the dagbag."""
+    dags = check_and_get_dags()
+
+    return dags
```

Comment:
A general comment:
I don't quite understand why you have function `get_dags()` here, which calls `check_and_get_dags()` then simply returns the result. When you call `get_dags()`, you can always directly use `check_and_get_dags()` instead. Then what's the idea of creating this `get_dags()`.

Reply:
I added the additional function/file for the sake of consistency in the project structure. Each API definition in `endpoints.py` calls some function in a file in `api/commons/experimental`. I agree that it's not necessary and I can get rid of it. I think it may be beneficial to refactor `api/commons/experimental`.